### DISCUSSION TITLE
Error reading preview.js when migrating to Storybook 8

### DISCUSSION THREAD

**Discussion by danyquestel** [storybookjs/storybook#26522] [Help]
### Summary

When trying to migrate my project from 7.6.10 to sb8, I got an error with the preview.jsx file.
I replace my preview.jsx with the one provided by the default one in the documentation https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#default-export-in-previewjs

But still I have the same error. Have I missed something in the migration ? what could be the problem ?

![image](https://github.com/storybookjs/storybook/assets/121247339/1a346ab9-d17a-4dfd-8395-3bdee5eae1fd)

![image](https://github.com/storybookjs/storybook/assets/121247339/920f042a-0d70-4fcc-b793-81c867c3167c)

**Comment by valentinpalkovic** [comment]
Hi @danyquestel 

Can you provide a minimal reproduction? https://storybook.new or by sharing a minimal reproduction repo.

**Comment by danyquestel** [comment]
Hi @valentinpalkovic ,
when working on building a reproductible repo I found the resolution of my problem. It was due to the following item mentionned in the migration doc.

***Removed @babel/core and babel-loader from @storybook/builder-webpack5***
In Storybook 8.0, we have removed the @storybook/builder-webpack5 package's dependency on Babel. This means that Babel is not preconfigured in @storybook/builder-webpack5. If you want to use Babel, you can add the necessary addon:
